ballerina/tests/resources/documents/block_strings_with_invalid_characters.txt

```diff
@@ -0,0 +1,7 @@
+{
+    sendEmail(message: """Hello,
+            World!,
+\"""
+        Yours,
+            GraphQL""") 
+}
```

Comment:
What is the invalid character in this?

Reply:
This test name should be ```testInvalidBlockStrings```. I will change it.